app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,73 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  contact: null,
+  tags: [],
 };
 
+export const CONTACTS = [
+  {
+    label: "Oliver Smith",
+    value: "Oliver",
+  },
+  {
+    label: "Sam Smith",
+    value: "Sam",
+  },
+  {
+    label: "Nancy Binani",
+    value: "Nancy",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Getting started",
+    value: "value1",
+  },
+  {
+    label: "Onboarding",
+    value: "value2",
+  },
+  {
+    label: "UX",
+    value: "value3",
+  },
+  {
+    label: "Bugs",
+    value: "value4",
+  },
+  {
+    label: "User flow",
+    value: "value5",
+  },
+  {
+    label: "V2",
+    value: "value6",
+  },
+];
+
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
+  contact: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string().oneOf(CONTACTS.map(contact => contact.label)),
+      value: yup.string().oneOf(CONTACTS.map(contact => contact.value)),
+    })
+    .required("Please select a contact"),
+  tags: yup
+    .array()
+    .of(
+      yup.object().shape({
+        label: yup.string().oneOf(TAGS.map(tag => tag.label)),
+        value: yup.string().oneOf(TAGS.map(tag => tag.value)),
+      })
+    )
+    .min(1, "Please select a tag")
```

Comment:
Here the error message can be made more specific. Here we should be pointing out that atleast one tag ought to be selected. That suggests two things:
- The least amount of tags needed
- More than one tag can also be selected